3515/2/helper_modules/auth.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'numbers_handler'
+require_relative 'check_this_campid_already_exist'
+require 'pry'
+
+module Auth
+  attr_reader :user_with_current_id
+
+  def already_registered?
+    @user_with_current_id = User.all.select { |user| user.id == from['id'].to_s }
+    !user_with_current_id.empty?
+  end
+
+  def already_registered_warning(camp_id)
+    respond_with :message, text: "Ваш аккаунт авторизован под номером #{camp_id} .\nДоступные
+    команды: /checkin, /checkout, /help"
+  end
+
+  def user_registration(*)
+    save_context :check_for_the_camp_id
+    respond_with :message, text: 'Вы еще не авторизованы. Введите свой camp_id...'
+  end
+
+  def check_for_the_camp_id(*words)
+    id = words[0].to_i
+    if NumbersHandler.offline_ids('data/numbers.yml').include?(id) &&
+       CheckTheCampidIsDefined.check(id)
```

Comment:
I think you don't need to create new module for only one check, you can do it in separate method

Reply:
I have to create new module CampIdCheckers and move these 2 checkers there, otherwise i got a warning from reek (doesn't depend on instance state (UtilityFunction))

Reply:
Moreover, to avoid this reek warning i have to call these methods like module methods, using self